src/main/scala/org/scalajs/tools/tsimporter/parser/TSDefParser.scala

```diff
@@ -112,6 +113,13 @@ class TSDefParser extends StdTokenParsers with ImplicitConversions {
   lazy val typeAliasDecl: Parser[DeclTree] =
     "type" ~> typeName ~ tparams ~ ("=" ~> typeDesc) <~ opt(";") ^^ TypeAliasDecl
 
+  lazy val importDecl: Parser[DeclTree] =
+    "import" ~> (
+        "{" ~ identifier ~ opt(lexical.Identifier("as") ~ identifier) ~ "}" ~ lexical.Identifier("from") ~ stringLiteral
```

Comment:
This is not general enough. It should be able parse multiple identifiers within the braces. For example:
```js
import { foo as bar, foobar, babar as baz } from "module.js"
```

Reply:
Should work now.